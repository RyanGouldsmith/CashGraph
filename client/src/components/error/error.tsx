import React from 'react';

export const Error = React.memo<{}>(() => {
  return (
    <React.Fragment>
      <h1>Sorry! That page does not exist</h1>
    </React.Fragment>
  );
});

export default Error;
