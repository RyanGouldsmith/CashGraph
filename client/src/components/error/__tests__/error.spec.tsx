import React from 'react';
import { render } from 'react-testing-library';

import { Error } from '../error';

test('Should render the error screen', () => {
  const errorComponent = render(<Error />);

  expect(errorComponent).toMatchSnapshot();
});
