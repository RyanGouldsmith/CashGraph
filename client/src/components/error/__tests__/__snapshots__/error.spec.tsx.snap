// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the error screen 1`] = `
Object {
  "baseElement": <body>
    <div>
      <h1>
        Sorry! That page does not exist
      </h1>
    </div>
  </body>,
  "container": <div>
    <h1>
      Sorry! That page does not exist
    </h1>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
